mqtt-kafka-connect/src/main/java/es/tid/kafnus/mqtt/MqttSourceTask.java

```diff
@@ -0,0 +1,151 @@
+package es.tid.kafnus.mqtt;
```

Comment:
es.tid. ? :)

Maybe we should use com.telefonica.

(Not only here but also in any place where we are currently using es.tid. in this PR)